rebuildall.sh

```diff
@@ -46,8 +46,7 @@ usage() {
   echo '          --no-motrkvscli-build    : Do not build motrkvscli tool, Default (false)'
   echo '          --no-s3background-build    : Do not build s3background process, Default (false)'
   echo '          --no-s3msgbus-build    : Do not build s3msgbus, Default (false)'
-  echo '          --no-s3recoverytool-build    : Do not build s3recoverytool process, Default (false)'
-  echo '          --no-s3confstoretool-build    : Do not build s3confstoretool process, Default (false)'
+#  echo '          --no-s3confstoretool-build    : Do not build s3confstoretool process, Default (false)'
```

Comment:
do not comment this. s3confstore has nothing to do with recovery tool

Reply:
it was by mistake. Done